schemas/telemetry/crash/crash.4.schema.json

```diff
@@ -1139,52 +1139,132 @@
         },
         "metadata": {
           "properties": {
+            "AsyncShutdownTimeout": {
+              "description": "<json>, Optional, present when a shutdown blocker failed to respond within a reasonable amount of time",
+              "type": "string"
+            },
             "AvailablePageFile": {
+              "description": "<size>, Windows-only, available paging file in bytes",
               "type": "string"
             },
             "AvailablePhysicalMemory": {
+              "description": "<size>, Windows-only, available physical memory in bytes",
               "type": "string"
             },
             "AvailableVirtualMemory": {
+              "description": "<size>, Windows-only, available virtual memory in bytes",
+              "type": "string"
+            },
+            "BlockedDllList": {
+              "description": "<list>, Windows-only, see WindowsDllBlocklist.cpp for details",
+              "type": "string"
+            },
+            "BlocklistInitFailed": {
+              "description": "1, Windows-only, present only if the DLL blocklist initialization failed",
               "type": "string"
             },
             "BuildID": {
+              "description": "YYYYMMDDHHMMSS",
+              "type": "string"
+            },
+            "ContainsMemoryReport": {
+              "description": "1, Optional, if set indicates that the crash had a memory report attached",
+              "type": "string"
+            },
+            "CrashTime": {
+              "description": "<time>, Seconds since the Epoch",
+              "type": "string"
+            },
+            "EventLoopNestingLevel": {
+              "description": "<levels>, Optional, present only if >0, indicates the nesting level of the event-loop",
               "type": "string"
             },
             "IsGarbageCollecting": {
+              "description": "1, Optional, if set indicates that the crash occurred while the garbage collector was running",
+              "type": "string"
+            },
+            "LowCommitSpaceEvents": {
+              "description": "<num>, Windows-only, present only if >0, number of low commit space events detected by the available memory tracker",
+              "type": "string"
+            },
+            "MemoryErrorCorrection": {
+              "description": "<type>, Windows-only, indicates the type of ECC memory in use, see below",
+              "type": "string"
+            },
+            "MozCrashReason": {
+              "description": "<reason>, Optional, contains the string passed to MOZ_CRASH()",
+              "type": "string"
+            },
+            "OOMAllocationSize": {
+              "description": "<size>, Size of the allocation that caused an OOM",
               "type": "string"
             },
             "ProductID": {
+              "description": "{ec8030f7-c20a-464f-9b0e-13a3a9e97384}",
```

Comment:
This doesn't look like a proper description to me. :) I guess it's just copypasted from the in-tree docs, would be nice to make that something more meaningful as well. Maybe at least we could file a bug?